Changelog.md

```diff
@@ -26,6 +26,7 @@ Language Features:
 
 Compiler Features:
  * Type Checker: Show named argument in case of error.
+ * Tests: Determine transaction status during IPC calls.
```

Comment:
In case anybody wonders: this is only in here as part of #4305, on which this PR depends and which is not yet merged.